ios/RNJudo.m

```diff
@@ -27,19 +30,39 @@ - (dispatch_queue_t)methodQueue
     return dispatch_get_main_queue();
 }
 
+- (NSDictionary *)constantsToExport
+{
+  return @{ @"APPLE_PAY_TRANSACTION_PAYMENT": @0,
+            @"APPLE_PAY_TRANSACTION_PREAUTH": @1,
+            @"PAYMENT_METHOD_NONE": @0,
+            @"PAYMENT_METHOD_CARD": @1,
+            @"PAYMENT_METHOD_APPLE_PAY": @2,
+            @"PAYMENT_METHOD_ALL": @3,
```

Comment:
Problematic having "ALL" as 3 from an extensibility point of view. Why not use String values for all these?

Reply:
I expose all the flags that are going from Judo SDK

Reply:
As tried to explain previously on the Java side, the "enum" values should be strings, not integers. Eg: `@"PAYMENT_METHOD_ALL": @"PAYMENT_METHOD_ALL"` - otherwise we will be in trouble when adding a new payment method